Report-template (#469)
github/securitylab: docs/report-template.md

I identified potential security vulnerabilities in [product].

I am committed to working with you to help resolve these issues. In this report you will find everything you need to effectively coordinate a resolution of these issues.

If at any point you have concerns or questions about this process, please do not hesitate to reach out to me at [email].

If you are _NOT_ the correct point of contact for this report, please let me know!

## Summary

*Short summary of the problem. Make the impact and severity as clear as possible. For example: An unsafe deserialization vulnerability allows any unauthenticated user to execute arbitrary code on the server.*

## Product

[product]

## Tested Version

[version]

## Details

*Give all details on the vulnerability. Pointing to the incriminated source code is very helpful for the maintainer.*

## PoC

*Complete instructions, including specific configuration details, to reproduce the vulnerability*

## Impact

[impact]

## Remediation

*Propose a remediation suggestion if you have one. Make it clear that this is just a suggestion, as the maintainer might have a better idea to fix the issue.*

## GitHub Security Advisories

> We recommend you create a private [GitHub Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/creating-a-security-advisory) for these findings. This also allows you to invite me to collaborate and further discuss these findings in private before they are [published](https://help.github.com/en/github/managing-security-vulnerabilities/publishing-a-security-advisory). I will be happy to collaborate with you, and review your fix to make sure that all corner cases are covered. 

Reviewer: I wonder if we should change this first sentence. "We recommend" makes sense when the report is coming from GitHub Security Lab, but maybe sounds a bit strange when coming from an independent security researcher. What about: "If possible, please could you create a private GHSA and invite me as a collaborator?"

Author: That's a good idea @kevinbackhouse thanks! I will do so

> We recommend you create a private [GitHub Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/creating-a-security-advisory) for these findings. This also allows you to invite me to collaborate and further discuss these findings in private before they are [published](https://help.github.com/en/github/managing-security-vulnerabilities/publishing-a-security-advisory). I will be happy to collaborate with you, and review your fix to make sure that all corner cases are covered. 

Author: ```suggestion
If possible, please could you create a private [GitHub Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/creating-a-security-advisory) for these findings? This allows you to invite me to collaborate and further discuss these findings in private before they are [published](https://help.github.com/en/github/managing-security-vulnerabilities/publishing-a-security-advisory). I will be happy to collaborate with you, and review your fix to make sure that all corner cases are covered. 
```

When you use a GitHub Security Advisory, you can request a CVE identification number from GitHub. GitHub usually reviews the request within 72 hours, and the CVE details will be published after you make your security advisory public. Publishing a GitHub Security Advisory and a CVE will help notify the downstream consumers of your project, so they can update to the fixed version.

## Credit

*List all researchers who contributed to this disclosure.*
*If you found the vulnerability with a specific tool, you can also credit this tool.*

## Contact

[contact]

## Disclosure Policy

> *Describe or link to your disclosure policy. It's important to have a disclosure policy where the public disclosure deadline, and the potential exceptions to it, are clear. You are free to copy and use the [GitHub Security Lab disclosure policy](https://securitylab.github.com/advisories/#policy).*

Reviewer: I expect most people who use this template will also be happy to use our disclosure policy. Can we put some standard language in here so that they don't need to edit it? (The thing that I am not sure about is whether the standard language can just link directly to our own disclosure policy, or whether we need them to make their own copy of it.)

## Disclosure Policy

*Describe or link to your disclosure policy. It's important to have a disclosure policy where the public disclosure deadline, and the potential exceptions to it, are clear. You are free to use the [GitHub Security Lab disclosure policy](https://securitylab.github.com/advisories/#policy), which is copied below for your convenience, if it resonates with you.*

> The *your_team_name_here* research team is dedicated to working closely with the open source community and with projects that are affected by a vulnerability, in order to protect users and ensure a coordinated disclosure. When we identify a vulnerability in a project, we will report it by contacting the publicly-listed security contact for the project if one exists; otherwise we will attempt to contact the project maintainers directly.

Reviewer 2: Should instances of "we" also be changed to "I" here as has been done above?

Reviewer 3: 6633c5d
